### PR TITLE
Computation run GUI bug fix and Run Comp standalone GUI Arguments

### LINKED ISSUE #821: "run computation" GUI, units of output values change when saving data
**Describe the bug**
When using the "run computation" GUI the units and labels for the output and input TimeSeries are set to those which are used within the computation algorithm which could either be the default of the units and labels which the user set to be input and output but could also be hardcoded defined units which are set within the algorithm. After the user executes their computation and clicks the "Save Output Data" button the units of only output time series which have labels hardcoded within the algorithm than that of which they are stored into the data base are then converted to the equivalent unit of that which is stored in the data base. While the units are correctly converted on save, the label of the units are never updated in the rendering, but the values of the data are.

**To Reproduce**
1. Go to the "run computation" GUI for your selected algorithm
2. Click execute
3. Click Save Output Data
4. if the units used within the algorithm are different that that saved to the database you will notice the values of the data to change but the label of the units will not change

**Expected behavior**
I would expect either the values of the data to never change and be re-rendered or for the label of the uniting to also change in the GUI

**Screenshots**
Before Saving Data-
runComputationGUI-beforeSave

After Saving Data-
runComputationGUI-afterSave

notice how the value of the data has changed for O:FTPK_Lower.EvapRate.Total.1hour but the label remains the same at "mm/hr"

**Desktop/ServerSide (please complete the following information):**
 - OS: Desktop Windows 10
 - Version: OpenDCS 7.0.14

**Additional context**

### PR DESCRIPTION
**Pull request by oskarhurst**
## Problem Description

Fixes #821

Describe the problem you are trying to solve.
TimeSeries table updates values of units but not labels of units on save button press in Comp Run GUI

## Solution

Describe your solution.
I set the table to completely re-render with new correct data and label on save button press.

secondary, added the feature where the user now has access to 3 new additional command line arguments -c "comp id or name" -S "since time where computation starts" -U "until time where computation ends". this match the structure of the compexec command. This is use full for faster testing so that the user does not have to reset parameters each time the GUI is opened.

## how you tested the change

Describe what was done to test the change. This section can be left blank 
if automated tests demonstrating usage are provided in the PR.

Testing UI components manually by running computation which have a data unit mismatch for output timeseries and insures timeSeries table updates correctly.

## Where the following done:

- [ ] Tests. Check all that apply:
   - [ ] Unit tests created or modified that run during ant test.
   - [ ] Integration tests created or modified that run during integration testing
         (Formerly called regression tests.)
   - [ ] Test procedure descriptions for manual testing
- [ ] Was relevant documentation updated?
- [ ] Were relevant config element (e.g. XML data) updated as appropriate

If you aren't sure leave unchecked and we will help guide you to want needs changing where.

do we want to update the legacy documentation for compexec?

### CODE REVIEW

File: java/opendcs/src/main/java/decodes/tsdb/comprungui/CompRunGuiFrame.java [hunk 1]
```diff
@@ -1074,6 +1131,8 @@ private void runSelectedComps(Collection<DbComputation> compVector, Logger origi
 			@Override
 			public List<CTimeSeries> doInBackground()
 			{
+				myinputs.clear();
```
Comment (MikeNeilson) [inline java/opendcs/src/main/java/decodes/tsdb/comprungui/CompRunGuiFrame.java:1134]: This should probably be around line 1129 with the myoutputs.clear(). Just for consistency.

File: java/opendcs/src/main/java/decodes/tsdb/comprungui/CompRunGuiFrame.java [hunk 2]
```diff
@@ -1135,6 +1194,7 @@ public List<CTimeSeries> doInBackground()
 							{
 								runme.addTimeSeries(ts);
 								inputs.add(ts);
+								myinputs.add(ts);
```
Comment (MikeNeilson) [inline java/opendcs/src/main/java/decodes/tsdb/comprungui/CompRunGuiFrame.java:1197]: I'm having a difficult time understanding why we need a 2nd array of the same inputs.

File: java/opendcs/src/main/java/decodes/tsdb/comprungui/RunComputationsFrameTester.java [hunk 3]
```diff
@@ -1,7 +1,14 @@
 package decodes.tsdb.comprungui;
```
Comment (MikeNeilson) [inline java/opendcs/src/main/java/decodes/tsdb/comprungui/RunComputationsFrameTester.java:1]: I know this existed already but would it make sense to just add the functionality to the main CompRunGuiFrame class?

**Review by MikeNeilson — CHANGES_REQUESTED** [review]

File: java/opendcs/src/main/java/decodes/tsdb/comprungui/RunComputationsFrameTester.java (hunk 3 above)
Comment (oskarhurst) [reply java/opendcs/src/main/java/decodes/tsdb/comprungui/RunComputationsFrameTester.java:1]: To my knowledge(which I may be wrong about) is that runComputationsFrameTester extends TsdbAppTemplate, where CompRunGuiFrame extends TopFrame. So CompRunGuiFrame does not have the capability to be run is a stand alone mode, so it needs the TsdbAppTemplate to initialized the gui then open it.

**Review by MikeNeilson — APPROVED** [review]

**Pull request merged**